src/clr-angular/data/datagrid/_datagrid.clarity.scss

```diff
@@ -871,29 +932,37 @@
       }
     }
 
-    .pagination-current {
-      font-weight: 600;
-      border-bottom: (2 * $clr-default-borderwidth) solid $action-blues-dark-midtone;
-    }
-
-    $clr-pagination-extra-margin: 0.25rem;
+    $clr-pagination-extra-margin: 0.5rem;
+    .pagination-first,
+    .pagination-last,
     .pagination-previous,
     .pagination-next {
-      display: inline-block;
-      vertical-align: middle;
+      display: flex;
+      align-items: center;
       width: $clr-datagrid-icon-size;
       height: $clr-datagrid-icon-size;
       background-repeat: no-repeat;
       background-size: contain;
     }
 
+    .pagination-first:disabled,
+    .pagination-last:disabled,
+    .pagination-previous:disabled,
+    .pagination-next:disabled {
+      clr-icon {
+        fill: #737373;
+        opacity: 0.54;
```

Comment:
generally speaking it's preferable to use a color value than opacity. opacity can have unintended side effects if users change the colors here via theming.

Reply:
After internal discussion, we are making `color` and `opacity` for disabled state both variables that can be overwritten via theming. I also moved out out of targeting `clr-icon` as per your suggestion.